Functions/Functions/User/AddUserOrganisationFunction.cs

```diff
@@ -45,8 +45,7 @@ public AddUserOrganisationFunction(RulesDbContext dbContext, IApiAuthorization a
             string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
             data = JsonConvert.DeserializeObject<User>(requestBody);
 
-            bool isNull = string.IsNullOrEmpty(data?.OrganisationId) || string.IsNullOrEmpty(data?.UserId);
-            if (data == null || isNull)
+            if (data == null || data?.OrganisationId == null || string.IsNullOrEmpty(data?.UserId))

```

Comment:
OrganisationId can never be null because it is not a nullable int. You should change organisationId to be a nullable int if that is a valid case. Otherwise, if that is not valid then remove the check for null.